nixos/modules/security/pam.nix

```diff
@@ -36,14 +36,61 @@ let
         '';
       };
 
-      u2fAuth = mkOption {
-        default = config.security.pam.enableU2F;
-        type = types.bool;
-        description = ''
-          If set, users listed in
-          <filename>~/.yubico/u2f_keys</filename> are able to log in
-          with the associated U2F key.
-        '';
+      U2F = {
```

Comment:
I'm big fan of "this-notation" or "this_notation" but I'm not sure we want that here. Some packages are using this convention, but I didn't see that for options.

Here I followed something like, camelCase but if the first letter is big I'm making it smaller just for naming sake. the same I would do if the name would started with API I wouldn't do apiSomething.

But I can use whatever convention is agreed :-)